canonical/ubuntu.com
# Add HWE to kernel lifecycle charts

## Done

- adds HWE to the kernel all releases chart
- adds relevant key
- changes how the kernel versions are attached to fix an alignment issue
- flyby: update colours for other release charts

## QA

- Go to https://ubuntu-com-13857.demos.haus/about/release-cycle#ubuntu-kernel-release-cycle
- See HWE has been added as per [the spreadsheet of truth](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=550510389)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10100

## Screenshots

Before :vomiting_face: 
![image](https://github.com/canonical/ubuntu.com/assets/58276363/ed016c86-57a5-447e-9832-ef42e49af129)

After :heart_eyes: 
![image](https://github.com/canonical/ubuntu.com/assets/58276363/626ffe93-dc0f-4217-97df-bff93178e6f6)



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
